lib/admin.js

```diff
@@ -18,6 +18,20 @@ const ENDPOINTS = {
     getFlaggedComments(offset) {
         return `/flagged-comments${offset ? `?offset=${offset}` : ''}`;
     },
+    setFlaggedShareReviewed(shareId) {
+        return {
+            path: `/flagged-shares/${shareId}/reviewed`,
+            method: 'PATCH',
+            responseType: 'text'
```

Comment:
are you sure this is the response type?

Reply:
yeh this shouldn't have a response type it's just the 200 status you get returned

Reply:
the response is empty. leaving it as default was throwing an error in the response data processing (as it was trying to parse JSON). this is how other empty responses are handled so added it

Reply:
hmmmmmmmmm v odd - kk do your thing 👍 ✨